packages/web/Dockerfile

```diff
@@ -2,7 +2,8 @@
 FROM mhart/alpine-node:9.4.0 AS base
 RUN apk add --update openrc nginx bash && rm -rf /var/cache/apk/*
 WORKDIR /app
-COPY web/nginx/default-vhost.conf /etc/nginx/conf.d/default.conf
+# COPY web/nginx/default-vhost.conf /etc/nginx/conf.d/default.conf
+# TODO: Copy yarn.lock
 COPY package.json yarn.lock /app/
 COPY api/package.json /app/api/package.json
 COPY core/package.json /app/core/package.json
```

Comment:
Nesting the dockerfiles is preferred the way you have it, should make sure that we can build these reliably since the default context of where the dockerfile is won't work, ie `docker build -f Dockerfile .`

Reply:
Yup. This is the same way CME did the multi-dockerfile-mono-repo